### ISSUE TITLE
if I run oneliner program, how can I get predicted probablity between 0 and 1, not a bining score(greater than 1)

### ISSUE THREAD

**Issue by 1512474508** [kexinhuang12345/DeepPurpose#33]

**Comment by kexinhuang12345** [comment]
One liner uses five pretrained models which are trained on BindingDB Kd scores. There are two ways: 1. To transform it by selecting a threshold: for example, below 10 is positive and above 100 is negative; or 2. Feed in your training dataset which has binary labels in one liner, it will automatically train five models.

**Comment by kexinhuang12345** [comment]
closing for now, feel free to reopen!
